actionview/lib/action_view/helpers/tags/select_renderer.rb

```diff
@@ -14,7 +14,7 @@ def select_content_tag(option_tags, options, html_options)
             add_default_name_and_id(html_options)
 
             if placeholder_required?(html_options)
-              raise ArgumentError, "include_blank cannot be false for a required field." if options[:include_blank] == false
+              raise ArgumentError, "include_blank cannot be false for the #{html_options["name"]} field, which is required." if options[:include_blank] == false
```

Comment:
Is there a chance name is blank?

Reply:
No,
in https://github.com/rails/rails/blob/main/actionview/lib/action_view/helpers/tags/select_renderer.rb we are adding the default name

Reply:
Anything else needed to be check here?